src/app/bulk-import/bulk-import.module.ts

```diff
@@ -0,0 +1,25 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+
+/** Custom Modules */
+import { SharedModule } from '../shared/shared.module'
+import { BulkImportRoutingModule } from './bulk-import-routing.module'
```

Comment:
https://travis-ci.com/openMF/web-app/builds/143895840

The Travis CI build is failing due to lint errors.